data/avro-schema/src/main/resources/avro/net/corda/data/uniqueness/UniquenessCheckExternalRequest.avsc

```diff
@@ -1,6 +1,6 @@
 {
   "type": "record",
-  "name": "UniquenessCheckRequest",
+  "name": "UniquenessCheckExternalRequest",
```

Comment:
I think it might be better to clearer to append `Avro` to all of these rather than use External, e.g. `UniquenessCheckRequestAvro`. This is accurate (they are Avro specific) and is a bit more future proof if we did use something other than Avro in future.